docs/docs-components/contexts/DocsExperimentProvider.js

```diff
@@ -8,7 +8,7 @@ import { ExperimentProvider } from 'gestalt';
  * - Unless you want the experimental behavior live on the docs for everyone, REMOVE YOUR EXPERIMENT HERE before merging your PR!
  * */
 
-const enabledExperiments = [];
+const enabledExperiments = ['web_gestalt_redesigned_toast', 'mweb_gestalt_redesigned_toast'];
```

Comment:
Same note here about not leaving these in until we provide the user a way to toggle them off.

Reply:
Moved to new logic